src/lib.rs

```diff
@@ -15,15 +18,41 @@ pub enum CharacterSet {
     UrlSafe
 }
 
+impl CharacterSet {
+    fn encode_table(&self) -> &'static [u8; 64] {
```

Comment:
I did this because I ended up duplicating those `match`es in the alternate encoding logic for the `Display` wrapper, but I think I actually like it even before that logic makes it an (aesthetic) necessity.